docs/tutorial/getting-started.rst

```diff
@@ -15,6 +15,10 @@ Prerequisites
 * Ubuntu 22.04 LTS.
 * An Internet connection.
 * `Snap`_ installed.
+* 50GB of disk space is recommended. This includes the `requirements <https://microk8s.io/docs/getting-started>`_ for MicroK8s.
```

Comment:
I added this info here, as part of the prerequisites.

Reply:
Looks good!